scanpipe/models.py

```diff
@@ -4345,3 +4348,122 @@ def create_auth_token(sender, instance=None, created=False, **kwargs):
     """Create an API key token on user creation, using the signal system."""
     if created:
         Token.objects.create(user_id=instance.pk)
+
+
+class DiscoveredPackageScore(UUIDPKModel, PackageScoreMixin):
+    """Represents a security or quality score for a DiscoveredPackage."""
+
+    discovered_package = models.ForeignKey(
+        DiscoveredPackage,
+        related_name="discovered_packages_score",
```

Comment:
`discovered_package.discovered_packages_score.all()` is a bit much.
I think a simple `scores` as related_name would be better:
`discovered_package.scores.all()`